src/Symfony/Component/Dotenv/Tests/DotenvTest.php

```diff
@@ -21,7 +21,7 @@ class DotenvTest extends TestCase
     /**
      * @dataProvider getEnvDataWithFormatErrors
      */
-    public function testParseWithFormatError($data, $error)
+    public function testParseWithFormatError(string $data, $error)
```

Comment:
```suggestion
    public function testParseWithFormatError(string $data, string $error)
```

Why did it ignored one argument?

Reply:
That's a good question, the `Tests` directory wasn't ignored or excluded, maybe Rector fails to build a type guesser for this node 🙁